react-client/src/Components/DomInfoDialogBox.js

```diff
@@ -122,25 +112,15 @@ const DomInfoDialogBox = ({ id, idx, clstag, clsname, parent, children, top, lef
                 <br />        
               </div>
             ))}         
-          </div>
-          {children.length - 1 > 2 ? (
-            leftover > 0 ? (
-              <div
-                id="closedompeeker"
-                className="see-more"
-                onClick={handleSeemore}
-              >
-                ... {leftover} more
-              </div>
-            ) : (
-              <div
-                id="closedompeeker"
-                className="see-more"
-                onClick={handleSeeless}
-              >
-                ... see less
-              </div>
-            )
+          </div>          
+          {children.length - 1 > 2 ? (            
```

Comment:
This logic can be improved further. If you want discussions I'll be happy to do so.